arduino-ide-extension/src/node/i18n/localization-backend-contribution.ts

```diff
@@ -0,0 +1,39 @@
+import * as express from 'express';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { LocalizationBackendContribution as TheiaLocalizationBackendContribution } from '@theia/core/lib/node/i18n/localization-backend-contribution';
+import { PluginDeployer } from '@theia/plugin-ext/lib/common/plugin-protocol';
+import { PluginDeployerImpl } from '@theia/plugin-ext/lib/main/node/plugin-deployer-impl';
+import { Deferred } from '@theia/core/lib/common/promise-util';
+
+@injectable()
+export class LocalizationBackendContribution extends TheiaLocalizationBackendContribution {
+  @inject(PluginDeployer)
+  protected readonly pluginDeployer: PluginDeployerImpl;
+
+  private initialized = new Deferred<void>();
+
+  override async initialize(): Promise<void> {
+    this.pluginDeployer.onDidDeploy(() => {
+      this.initialized.resolve();
+    });
+    await this.localizationRegistry.initialize();
```

Comment:
Why not calling `super.initialize()`. The default Theia behavior is what IDE2 wants to use besides the listener.